packages/core/database/lib/index.d.ts

```diff
@@ -139,6 +139,7 @@ export interface Database {
   entityManager: EntityManager;
 
   query<T extends keyof AllTypes>(uid: T): QueryFromContentType<T>;
+  transaction(cb: (em: EntityManager) => Promise<any>): Promise<void>;
```

Comment:
would be useful if the `transaction` method returned `Promise<any>` so that created entity could be conveniently returned from the transaction.

```diff
-transaction(cb: (em: EntityManager) => Promise<any>): Promise<void>;
+transaction(cb: (em: EntityManager) => Promise<any>): Promise<any>;
```